### ISSUE TITLE
the windows package has a redundunt artifact

### ISSUE THREAD

**Issue by garfield69** [FlareSolverr/FlareSolverr#745]
### Have you checked our README?

- [X] I have checked the README

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: 3.1.1
- Last working FlareSolverr version:
- Operating system: windows
- Are you using Docker: [yes/no]
- FlareSolverr User-Agent (see log traces or / endpoint):
- Are you using a VPN: [yes/no]
- Are you using a Proxy: [yes/no]
- Are you using Captcha Solver: [yes/no]
- If using captcha solver, which one:
- URL to test this issue:
```

### Description

minor finding,  tidy up at your leasure.
when unpacking the windows 3.1.1 flaresolverr package (flaresolverr_windows_x64.zip), the extract creates an additional flaresolverr_windows_x64.zip file along with the expected flaresolverr folder.
that additional zip file appears invalid or is empty.

### Logged Error Messages

```text
not relevant
```

**Comment by ngosang** [comment]
I will fix this issue in the next release. Nothing to worry about. Thank you for the report!

**Comment by jmackxiii** [comment]
Just curious if the windows format will be in exe or zip? last release was exe now, this is zip. Trying to update my auto updated script and do not want to modify it again if it will revert back to exe.

**Comment by ngosang** [comment]
It will be zip. I had to make the change because it was causing issues for some users => https://github.com/FlareSolverr/FlareSolverr/issues/741

**Comment by jmackxiii** [comment]
Thank you.. I also noticed there was a password when exporting the nested zip file.. is that going to be the case going forward.. or will that nested zip folder be a exe and not password protected?

**Comment by ngosang** [comment]
The nested zip was a bug, I will remove it. You only need to uncompress the zip and there will be just one folder called "flaresolverr".

**Comment by jmackxiii** [comment]
Got it.. will the exe be in there? I'll keep an eye out for it and modify my auto updated according. Thank you very much.

**Comment by ngosang** [comment]
> Got it.. will the exe be in there?

Sure, inside the folder.

**Comment by jmackxiii** [comment]
Thank you.

**Comment by ngosang** [comment]
Fixed in ac77110578f49874c7e31f261cf88375c681d6c8
